SConstruct

```diff
@@ -115,6 +115,7 @@ opts.Add(EnumVariable("target", "Compilation target", "debug", ("debug", "releas
 opts.Add(EnumVariable("optimize", "Optimization type", "speed", ("speed", "size")))
 
 opts.Add(BoolVariable("tools", "Build the tools (a.k.a. the Godot editor)", True))
+opts.Add(BoolVariable("tests", "Build the unit tests", True))
```

Comment:
I think `tests` should be only enabled on `tools` builds, I don't think they would work on export templates anyway as is.

And maybe the default value should be `False` for end users who build the engine, and make `dev=yes` enable it.